test/utils.js

```diff
@@ -3,6 +3,8 @@
 
 'use strict';
 
+/* eslint no-control-regex: 0 */
```

Comment:
Could this be replaced by `// eslint-disable-next-line no-control-regex` right before the `INVISIBLES_REGEXP` (I didn't see other RegExps with control characters in this file.)

Reply:
I don't know why but I couldn't get it to work before, now I'm having no trouble with it, odd...